misc_docs/syntax/language_polyvar.mdx

```diff
@@ -0,0 +1,39 @@
+---
+id: "polyvar"
+keywords: ["polymorphic", "variant", "polyvar"]
+name: "[ #value ]"
+summary: "This is the `polymorphic variant` type declaration."
+category: "languageconstructs"
+---
+
+[Polymorphic variants](/docs/manual/latest/polymorphic-variant) (or _poly variants_) are a specific kind of [variant](/docs/manual/latest/variant).
+
+Their usage doesn't require any explicit type definition, and they not coupled to any specific module. The compiler will infer the type on demand, and compare poly variants by their value, instead of their type name.
```

Comment:
```suggestion
In comparison to nominally typed variants, their values don't require any explicit type definition, and are not coupled to any specific module. The compiler will infer the type on demand, and compare poly variants by their value, instead of their type name.
```